.github/workflows/r_nightly.yml

```diff
@@ -25,7 +28,7 @@ on:
         required: false
         default: ''
   schedule:

```

Comment:
Should we also trigger this workflow on changes to the workflow file itself, like we do for other workflows (e.g. https://github.com/apache/arrow/blob/master/.github/workflows/r.yml#L23)? There are tradeoffs but it would make it easier to test changes to the nightly uploading code, which is non-trivial.

Reply:
I didn't add it because we have the manual trigger but it would also not do anything worse than re-upload the most recent packages so we could add it.

Reply:
What is the manual trigger?

Reply:
If you have the correct access you can go [here](https://github.com/apache/arrow/actions/workflows/r_nightly.yml) and run it with a prefix of your choice (e.g. one from a PR like `actions-b618126b5b`). Which should be helpful to upload the release binaries without having to edit a workflow.

Reply:
I don't quite understand what I would do, can you document this somewhere? Perhaps on the release management job?

Reply:
Or maybe there's a followup issue that is needed to add this to the release prep workflow?